### PR TITLE
Feature/single game

### LINKED ISSUE #3: Create single game route and view
As a user I want a page where I can view a game's information, such as description, rating, complexity, etc.

Acceptance Criteria:
- As mentioned in #2 we need a GameController to handle routes pertaining to games
- There is a route called /game/{id} that takes a game id parameter
- There is a view called game-info loaded by the above route
- We can get to this screen from the game list view
- For now some static placeholder game info will suffice

### PR DESCRIPTION
**Pull request by mihaelavalac**
Completed  #3

### CODE REVIEW

File: resources/views/layouts/navigation.blade.php [hunk 1]
```diff
@@ -12,18 +12,38 @@
...
                         {{ __('Game List') }}
                     </x-nav-link>
+
+                    <div class="mb-2 mt-3 ml-3 xl:w-96">
+                        <div class="input-group relative flex flex-wrap items-stretch w-full mb-4">
+                            <input type="search" class="form-control relative flex-auto min-w-0 block w-full px-3 py-1.5 text-base font-normal text-black-700 bg-white bg-clip-padding border border-solid border-gray-500 rounded transition ease-in-out m-0 focus:text-black-700 focus:bg-white focus:border-blue-600 focus:outline-none" placeholder="Search" aria-label="Search" aria-describedby="button-addon2">
+
+                            <x-nav-link :href="route('results')" :active="request()->routeIs('search-results')" ><button class="btn inline-block px-6 py-4 bg-black-600 text-black font-medium text-xs leading-tight uppercase rounded shadow-md hover:bg-green-300 hover:shadow-lg focus:bg-blue-700  focus:shadow-lg focus:outline-none focus:ring-0 active:bg-blue-800 active:shadow-lg transition duration-150 ease-in-out flex items-center" type="button" id="button-addon2">
```
Comment (benwagner813) [inline resources/views/layouts/navigation.blade.php]: The search button looks great! I think we need to put the search bar and button inside a form element and change the button to submit.

**Review by benwagner813 — DISMISSED** [review]
Some small changes needed but otherwise nothing breaks. I realize now that we haven't updated the responsive nav. The search bar and the game list both need to be added.

**Comment by benwagner813** [comment]
I decided to make some changes since I noticed something I forgot to include. Along the way I messed up the search button(pic for reference). I spent a while trying to fix it but I couldn't manage. I really struggle with css sometimes, any tips are appreciated.
mistake

**Review by mihaelavalac — COMMENTED** [review]
I will work on the search box to fix the issue.

**Review by ryvrook — APPROVED** [review]
Single Game w/ Game placeholders added. Looks good, and working.

**Pull request merged**